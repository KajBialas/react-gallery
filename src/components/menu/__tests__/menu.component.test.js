import React from 'react';
import { shallow } from 'enzyme';
import MenuComponent from '../menu.component';

describe('MenuComponent', () => {
  it('should shallows correctly', () => {
    const component = shallow(<MenuComponent />);

    expect(component).toMatchSnapshot();
  });
});