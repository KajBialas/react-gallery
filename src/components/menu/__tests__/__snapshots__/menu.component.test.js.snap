// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuComponent should shallows correctly 1`] = `
<styled.div>
  <span />
  <span />
  <span />
  <span />
  <span />
  <styled.ul>
    <styled.li
      key="0"
    >
      <Styled(NavLink)
        count={1}
        to="/"
      >
        Home
      </Styled(NavLink)>
    </styled.li>
    <styled.li
      key="1"
    >
      <Styled(NavLink)
        count={2}
        to="/favorites"
      >
        Favorites
      </Styled(NavLink)>
    </styled.li>
    <styled.li
      key="3"
    >
      <Styled(NavLink)
        count={3}
        to="/contact"
      >
        Contact
      </Styled(NavLink)>
    </styled.li>
  </styled.ul>
</styled.div>
`;
